flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java

```diff
@@ -49,14 +49,17 @@ public void pointTo(MemorySegment[] segments, int offset, int sizeInBytes) {
         this.sizeInBytes = sizeInBytes;
     }
 
+    @Override
```

Comment:
I don't think these changes are relevant to the hotfix doc PR?

Reply:
These methods are missing @Override annotations.  I added

Reply:
These require a new PR or do not need to be fixed ?

Reply:
I'm not sure the methods are missing and I would remove them from a hotfix PR on documentation

Reply:
The @Override annotation modifications have been removed in this pr